### PR TITLE
Fix diamond tests

### PR DESCRIPTION
**Pull request by gnufied**
This enables running of collector tests in CI workflow.

Tests that have not been passing or erroring out are blacklisted.

### CODE REVIEW

**Comment by tiras-j** [comment]
lgtm! 

Btw I'd make a wild guess that quite a few failing tests are due to our changes with cumulative counters (since before diamond did the delta calculations locally) and names etc. Of course working through all those just to fix em up would be quite tedious.

**Comment by gnufied** [comment]
Cumulative counters and we added support for dimensions.  

I agree that fixing all of these would be quite tedious but I think we can do this as we modify/touch an individual collector. Good news is - about 50% of collector tests are passing and working. So at least -  we make sure we are running tests that are good as part of the build process.

File: Makefile [hunk 1]
```diff
@@ -59,7 +59,7 @@ $(BEATIT): $(BEATIT_SOURCES)
 	@gom build -o bin/$(BEATIT) $@
 
 test: tests
-tests: deps diamond_core_test
+tests: deps diamond_core_test diamond_collecor_test
```
Comment (poros) [inline Makefile]: typo: collector

**Comment by poros** [comment]
+1

**Pull request merged**